diff --git a/node_modules/@cowprotocol/sdk-config/dist/index.d.ts b/node_modules/@cowprotocol/sdk-config/dist/index.d.ts
index 3c0b8f7..aea86c3 100644
--- a/node_modules/@cowprotocol/sdk-config/dist/index.d.ts
+++ b/node_modules/@cowprotocol/sdk-config/dist/index.d.ts
@@ -27,10 +27,10 @@ declare enum SupportedChainId {
     POLYGON = 137,
     LENS = 232,
     BASE = 8453,
-    PLASMA = 9745,
+    // PLASMA = 9745,
     ARBITRUM_ONE = 42161,
     AVALANCHE = 43114,
-    LINEA = 59144,
+    // LINEA = 59144,
     SEPOLIA = 11155111
 }
 /**
diff --git a/node_modules/@cowprotocol/sdk-config/dist/index.js b/node_modules/@cowprotocol/sdk-config/dist/index.js
index d986305..106c134 100644
--- a/node_modules/@cowprotocol/sdk-config/dist/index.js
+++ b/node_modules/@cowprotocol/sdk-config/dist/index.js
@@ -769,8 +769,8 @@ var ALL_SUPPORTED_CHAINS_MAP = {
   [137 /* POLYGON */]: polygon,
   [56 /* BNB */]: bnb,
   [232 /* LENS */]: lens,
-  [9745 /* PLASMA */]: plasma,
-  [59144 /* LINEA */]: linea,
+  // [9745 /* PLASMA */]: plasma,
+  // [59144 /* LINEA */]: linea,
   [11155111 /* SEPOLIA */]: sepolia
 };
 var ALL_SUPPORTED_CHAINS = Object.values(ALL_SUPPORTED_CHAINS_MAP);
diff --git a/node_modules/@cowprotocol/sdk-config/dist/index.mjs b/node_modules/@cowprotocol/sdk-config/dist/index.mjs
index fdc23e0..c2b97fb 100644
--- a/node_modules/@cowprotocol/sdk-config/dist/index.mjs
+++ b/node_modules/@cowprotocol/sdk-config/dist/index.mjs
@@ -696,8 +696,8 @@ var ALL_SUPPORTED_CHAINS_MAP = {
   [137 /* POLYGON */]: polygon,
   [56 /* BNB */]: bnb,
   [232 /* LENS */]: lens,
-  [9745 /* PLASMA */]: plasma,
-  [59144 /* LINEA */]: linea,
+  // [9745 /* PLASMA */]: plasma,
+  // [59144 /* LINEA */]: linea,
   [11155111 /* SEPOLIA */]: sepolia
 };
 var ALL_SUPPORTED_CHAINS = Object.values(ALL_SUPPORTED_CHAINS_MAP);
