diff --git a/node_modules/@cowprotocol/sdk-config/dist/index.d.ts b/node_modules/@cowprotocol/sdk-config/dist/index.d.ts
index 3c0b8f7..eb55839 100644
--- a/node_modules/@cowprotocol/sdk-config/dist/index.d.ts
+++ b/node_modules/@cowprotocol/sdk-config/dist/index.d.ts
@@ -27,10 +27,10 @@ declare enum SupportedChainId {
     POLYGON = 137,
     LENS = 232,
     BASE = 8453,
-    PLASMA = 9745,
+    //PLASMA = 9745,
     ARBITRUM_ONE = 42161,
     AVALANCHE = 43114,
-    LINEA = 59144,
+    //LINEA = 59144,
     SEPOLIA = 11155111
 }
 /**
