diff --git a/node_modules/@web3-react/metamask/dist/index.js b/node_modules/@web3-react/metamask/dist/index.js
index 38433a2..463f580 100644
--- a/node_modules/@web3-react/metamask/dist/index.js
+++ b/node_modules/@web3-react/metamask/dist/index.js
@@ -67,6 +67,10 @@ class MetaMask extends types_1.Connector {
                         this.actions.update({ chainId: parseChainId(chainId) });
                     });
                     this.provider.on('disconnect', (error) => {
+                        if (error.code === 1013) {
+                            console.log('Ignore the error (1013): "MetaMask: Disconnected from chain. Attempting to connect.:"', error);
+                            return;
+                        }
                         var _a;
                         this.actions.resetState();
                         (_a = this.onError) === null || _a === void 0 ? void 0 : _a.call(this, error);
