diff --git a/node_modules/@web3-onboard/walletconnect/dist/index.d.ts b/node_modules/@web3-onboard/walletconnect/dist/index.d.ts
index a02f819..053e739 100644
--- a/node_modules/@web3-onboard/walletconnect/dist/index.d.ts
+++ b/node_modules/@web3-onboard/walletconnect/dist/index.d.ts
@@ -5,6 +5,8 @@ interface WalletConnectOptions {
         mobileLinks: string[];
     };
     connectFirstChainId?: boolean;
+    getIcon?: () => any;
+    label?: string;
 }
 declare function walletConnect(options?: WalletConnectOptions): WalletInit;
 export default walletConnect;
diff --git a/node_modules/@web3-onboard/walletconnect/dist/index.js b/node_modules/@web3-onboard/walletconnect/dist/index.js
index c122028..71c41a5 100644
--- a/node_modules/@web3-onboard/walletconnect/dist/index.js
+++ b/node_modules/@web3-onboard/walletconnect/dist/index.js
@@ -8,8 +8,14 @@ function walletConnect(options) {
     const { bridge = 'https://bridge.walletconnect.org', qrcodeModalOptions, connectFirstChainId } = options || {};
     return () => {
         return {
-            label: 'WalletConnect',
-            getIcon: async () => (await import('./icon.js')).default,
+            label: options.label || 'WalletConnect',
+            getIcon: async () => {
+                if (options.getIcon) {
+                    return options.getIcon()
+                }
+
+                return (await import('./icon.js')).default
+            },
             getInterface: async ({ chains, EventEmitter }) => {
                 const { StaticJsonRpcProvider } = await import('@ethersproject/providers');
                 const { ProviderRpcError, ProviderRpcErrorCode } = await import('@web3-onboard/common');
