// @uniswap/v3-periphery/artifacts/contracts/interfaces/index.ts
/* Autogenerated file. Do not edit manually. */
/* tslint:disable */
/* eslint-disable */
import type * as external from "@uniswap/v3-periphery/artifacts/contracts/interfaces/external";
export type { external };
export type { IERC20Metadata } from "@uniswap/v3-periphery/artifacts/contracts/interfaces/IERC20Metadata";
export type { IERC721Permit } from "@uniswap/v3-periphery/artifacts/contracts/interfaces/IERC721Permit";
export type { IMulticall } from "@uniswap/v3-periphery/artifacts/contracts/interfaces/IMulticall";
export type { INonfungiblePositionManager } from "@uniswap/v3-periphery/artifacts/contracts/interfaces/INonfungiblePositionManager";
export type { INonfungibleTokenPositionDescriptor } from "@uniswap/v3-periphery/artifacts/contracts/interfaces/INonfungibleTokenPositionDescriptor";
export type { IPeripheryImmutableState } from "@uniswap/v3-periphery/artifacts/contracts/interfaces/IPeripheryImmutableState";
export type { IPeripheryPayments } from "@uniswap/v3-periphery/artifacts/contracts/interfaces/IPeripheryPayments";
export type { IPeripheryPaymentsWithFee } from "@uniswap/v3-periphery/artifacts/contracts/interfaces/IPeripheryPaymentsWithFee";
export type { IPoolInitializer } from "@uniswap/v3-periphery/artifacts/contracts/interfaces/IPoolInitializer";
export type { IQuoter } from "@uniswap/v3-periphery/artifacts/contracts/interfaces/IQuoter";
export type { IQuoterV2 } from "@uniswap/v3-periphery/artifacts/contracts/interfaces/IQuoterV2";
export type { ISelfPermit } from "@uniswap/v3-periphery/artifacts/contracts/interfaces/ISelfPermit";
export type { ISwapRouter } from "@uniswap/v3-periphery/artifacts/contracts/interfaces/ISwapRouter";
export type { ITickLens } from "@uniswap/v3-periphery/artifacts/contracts/interfaces/ITickLens";
export type { IV3Migrator } from "@uniswap/v3-periphery/artifacts/contracts/interfaces/IV3Migrator";
